packages/os/bootstrap-containers-toml

```diff
@@ -0,0 +1,16 @@
+[required-extensions]
+bootstrap-containers = "v1"
++++
+{{#if settings.bootstrap-containers}}
+{{#each settings.bootstrap-containers}}
+["{{@key}}"]
+source = "{{this.source}}"
+mode = "{{this.mode}}"
+{{#if this.user-data}}
+user-data = "{{this.user-data}}"
```

Comment:
this will URL-escape the special characters, most importantly the `==` at the end of the base64 string. In order to avoid this, you need to format it with triple curly braces: https://handlebarsjs.com/guide/#block-helpers

(ran into this while testing host-containers 😄)

```suggestion
user-data = "{{{this.user-data}}}"
```